fenix/benchmark/src/main/java/org/mozilla/fenix/benchmark/utils/Constants.kt

```diff
@@ -0,0 +1,4 @@
+package org.mozilla.fenix.benchmark.utils
+
+const val TARGET_PACKAGE = "org.mozilla.fenix"
+const val DEFAULT_ITERATIONS = 5
```

Comment:
Could these files be moved into the test source tree or is there a reason they need to be in the main source tree?

Reply:
This is in the `benchmarks` module, which is like `androidTests` and don't go into the compiled main apk. AFAIK benchmark installs it's own apk that controls the main apk using ui automator.

Reply:
Ah I missed that. Thanks!